MyCryptoHQ/MyCrypto
# [BUGFIX] Do not switch off scheduling toggle when token has been selected

### Description

Don't switch off scheduling toggle when token has been selected.

This code that now is being removed has been introduced, when we wanted to disable scheduling for token transfers, but since now it's been implemented, there's no need for toggling scheduling off for tokens anymore.

### Steps to Test

1. Select ETH
2. Select Send Later
3. Select any token (something different than ETH)

Expected: Send Later toggle should remain enabled.
